#!/usr/bin/env python
PACKAGE = "cascade_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("K_pxy",    double_t, 0, "K_pxy",       3.3, 1,   5)
gen.add("K_pz",     double_t, 0, "K_pz",        3.5, 1,   5)
gen.add("K_vxy",    double_t, 0, "K_vxy",       2.5, 1,   5)
gen.add("K_vz",     double_t, 0, "K_vz",        2.8, 1,   5)

exit(gen.generate(PACKAGE, "cascade_controller", "params"))
